lib/Mojo/WebSocketProxy.pm

```diff
@@ -128,6 +128,48 @@ supported.
 
 Alternate backends are selected by using the C<backend> action option.
 
+=head2 rpc_failure_cb
+
+A sub routine reference to call if the remote call fails at a http level. 
+Called with C<< Mojolicious::Controller >> the rpc_response 
+and C<< $req_storage >>
+
+A default rpc_failure_cb could be provided in the startup sub routine
+
+    sub startup {
+        my $self = shift;
+        $self->plugin(
+            'web_socket_proxy' => {
+                actions => [
+                    ['json_key', {some_param => 'some_value'}]
+                ],
+                base_path => '/api',
+                url => 'http://rpc-host.com:8080/',
+                rpc_failure_cb => sub {
+                    my ($c, $res, $req_storage) = @_;
+                    warn "Something went wrong an RPC call";
```

Comment:
```suggestion
                    warn "RPC call failed";
```

By this point, `$res` should have something that could indicate the kind of error encounted (e.g. HTTP status code...) since we know further down that `$res->is_error`...